src/barnetilsyn/steg/6-barnepass/Barnepass.tsx

```diff
@@ -174,7 +192,12 @@ const Barnepass: FC<Props> = () => {
             søkerFraBestemtMåned={søkerFraBestemtMåned}
             valgtDato={søknadsdato}
             datovelgerLabel={datovelgerLabel}
-            hjelpetekstInnholdTekst={hjelpetekstInnholdSøkerFraMndTekst}
+            hjelpetekstInnholdTekst={
+              <StyledHjelpetekst>
+                <BodyShort>{hjelpetekstInnholdSøkerFraMndTekst}</BodyShort>
```

Comment:
```suggestion
                <BodyShort>{hjelpetekstInnholdSøkerFraMndTekstDel1}</BodyShort>
```
Bare litt småpirk :p

Reply:
👍